java/src/main/java/com/training/db/hw2/Algorithm.java

```diff
@@ -0,0 +1,22 @@
+package com.training.db.hw2;
+
+public class Algorithm {
+
+    public static void main(String[] args) {
+
+        int n1 = UiUtil.input("Input first number: ");
+        int n2_fac = UiUtil.input("Input second number: ");
```

Comment:
1) in java we use camalCase instead of snake_case, the only exception for snake_case is static constants. For more details about code styling you can read in java code convention.
2) it's no need for create 2 different n2 for fac and fib, you can use 1 common variable and pass it to function

Reply:
done